activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/converter/EventConverterContext.java

```diff
@@ -85,6 +87,15 @@ public static String getPrefix(ActivitiEvent activitiEvent) {
                     return "ProcessInstance:";
                 } else if (entity instanceof Task) {
                     return "Task:";
+                } else if (entity instanceof IdentityLink){
```

Comment:
We'll need to think (not in this PR) about a better solution for this getPrefix method. It's becoming too complex.